osie-runner/run.py

```diff
@@ -119,7 +119,7 @@ def connect_hegel(facility):
     log.info("about to monitor")
     try:
         resp = watch.next()
-    except grpc.RpcError as e:
+    except (grpc.RpcError, grpc._channel._Rendezvous) as e:
         log.exception("grpc error")
```

Comment:
Hmm, I don't think this will change the level of verbosity. If an exception was being raise that wasn't being caught, it would fail out of this while loop and exit. Where as currently it prints the exception and retries.

I'm thinking perhaps this `log.exception` line is probably printing the exception along with the provided message. So if we're trying to get rid of the exception output, we might just need to remove that line.